sensors/shtc3Sensor/SHTC3Sensor.cpp

```
@@ -0,0 +1,117 @@
+}
+
+void SHTC3Sensor::setMeasurementInterval(uint32_t interval_ms) {
+    measurement_interval_ms_ = interval_ms;
+}
+
+void SHTC3Sensor::setMeasurementCallback(void (*callback)(int32_t temperature, int32_t humidity)) {
+    measurement_callback_ = callback;
+}
+
+void SHTC3Sensor::startContinuousMeasurement() {
+    if (continuous_active_) return;
```

Clang (14.0.6)

[warning] statement should be inside braces

(readability-braces-around-statements)